doc/admin-api-cn.md

```diff
@@ -71,7 +75,29 @@
 - 除了 `uri`/`uris` 是必选的之外，`plugins`、`upstream`/`upstream_id`、`service_id` 这三类必须选择其中至少一个。
 - 对于同一类参数比如 `uri`与 `uris`，`upstream` 与 `upstream_id`，`host` 与 `hosts`，`remote_addr` 与 `remote_addrs` 等，是不能同时存在，二者只能选择其一。如果同时启用，接口会报错。
 
-示例：
+route 对象 json 配置内容：
+
+```json
```

Comment:
we should use ```shell here.

Reply:
![image](https://user-images.githubusercontent.com/6814606/76692845-241a0000-6697-11ea-9b89-ed0a3b6ec81d.png)

Reply:
Why is "```shell" better?
And I have corrected it.

Reply:
![image](https://user-images.githubusercontent.com/6814606/76703265-7432aa00-670b-11ea-8a41-4370acda4d4e.png)